string  trigger_selector    # Trigger selector
---
